test/unit/array.spec.ts

```diff
@@ -191,3 +191,21 @@ test.each([
     `;
     expect(() => util.transpileAndExecute(code)).toThrowError(`invalid array length: ${result}`);
 });
+
+test("Array OmittedExpression", () => {
+    const result = util.transpileAndExecute(
+        `const myarray = [,];
```

Comment:
I'd prefer it if you kept the array the same in the two tests, so check index 1  on [1,,2] for this one.

Reply:
I've used a `test.each` to check that array's contents